### DISCUSSION TITLE
Safari Extension

### DISCUSSION THREAD

**Discussion by thekendog** [Dhruv-Techapps/acf-docs#359] [Polls]
Couldn't find any other way to ask this (no contact form on site and no messaging on Twitter enabled), so I apologize if this is not the right place.  Is there any plans or possibility to get this extension ported to Safari?  In the process of switching and this is my most used extension and I miss it BADLY.

**Comment by dharmesh-hemaram** [comment]
We will check on this. Currently, we don't have a plan to migrate to safari as we have a windows system only. Safari for windows is totally different from Safari on MAC

**Comment by Dhruv-Tech-Apps** [comment]
Please upvote here based on the number of user responses we will provide extensive support for other browsers.
